client/webpack.config.js

```diff
@@ -1,3 +1,8 @@
+/**
+ * TODO: Split up development configuration from production configuration
```

Comment:
it is a bit more involved than that sadly, we will need to move react_on_rails to v9 and bump us up to webpacker instead of webpacker_lite gem